eo-parser/src/test/resources/org/eolang/parser/packs/add-probes.yaml

```diff
@@ -1,13 +1,21 @@
+xsls:
+  - /org/eolang/parser/errors/not-empty-atoms.xsl
```

Comment:
@maxonfjvipon Maybe it's better to create a separate test file for this functionality? What do you think?

Reply:
@volodya-lombrozo you mean `not-empty-atoms`?

Reply:
@volodya-lombrozo this file did not have `xsls` section at all (that means that all xsl transformations are applied). I added this section just because `add-probes` xsl goes after `wrong-versions` which fails for now.  I believe we can just leave `add-probes` xsl here

Reply:
@maxonfjvipon I meant the next: Instead of adding `xsls` section into `add-probes.yaml`, can we just create one more test pack like `add-probes-versioned.yaml`?

Reply:
@volodya-lombrozo ok, let's do it